pkg/cloudprovider/providers/openstack/metadata.go

```diff
@@ -132,14 +129,12 @@ func getMetadataFromMetadataService() (*Metadata, error) {
 	glog.V(4).Infof("Attempting to fetch metadata from %s", metadataUrl)
 	resp, err := http.Get(metadataUrl)
 	if err != nil {
-		glog.V(3).Infof("Cannot read %s: %v", metadataUrl, err)
```

Comment:
Should probably include this context, since (eg) a "Permission Denied" error is going to look very confusing without some indication that it received that error while trying to look up a URL.

```go
return nil, fmt.Errorf("Error fetching %s: %v", metadataUrl, err)
```

Reply:
yeah, done